zebra/explorer.md

```diff
@@ -13,23 +13,23 @@ https://zcashblockexplorer.com/
-| z_validateaddress           | NO.                | Same as `validateaddress`                                                                                                                                                                                           | [Zcash](https://zcash.github.io/rpc/z_validateaddress.html)                                                                                                                  | [1](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/search_controller.ex#L17)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| z_validatepaymentdisclosure | NO.                | Need to be fully implemented. Specification still in draft.                                                                                                                           | [Zcash](https://zcash.github.io/rpc/z_validatepaymentdisclosure.html) [docs](https://github.com/zcash/zcash/blob/master/doc/payment-disclosure.md) [draft ZIP](https://github.com/zcash/zips/pull/119)                                                                                                       | [1](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/page_controller.ex#L49)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
This is a draft ZIP, maybe we could just skip this RPC? It hasn't been updated for Orchard yet, and it needs some format changes to work with NU5.